src/services/SocketService/socketErrorService.js

```diff
@@ -0,0 +1,66 @@
+import camelcaseKeys from 'camelcase-keys';
+import store from 'app/store';
+import { errChat } from 'reducers/ChatDux';
+import Socket from './socketSetup';
+
+export const errChatRooms = () => {
+  Socket.socket.on('err_chat_rooms', payload => {
```

Comment:
I suggest keeping all the socket channel constants in `constants/socket.js`:

```
// constants/socket.js
const CHATROOM_ERROR = 'err_chat_rooms';
...
```
```
// here, socketErrorService.js
export const errChatRooms = () => {
  Socket.socket.on(CHATROOM_ERROR, payload => {
...
}
```

Makes it much easier to read